exec/jdbc/pom.xml

```diff
@@ -134,6 +134,7 @@
         <dependency>
           <groupId>org.apache.hadoop</groupId>
           <artifactId>hadoop-common</artifactId>
+          <scope>test</scope>
```

Comment:
hadoop-common is not a dependency for jdbc or jdbc-all project. [drill-jdbc-all-1.5.0.jar](https://repo.maven.apache.org/maven2/org/apache/drill/exec/drill-jdbc-all/1.5.0/drill-jdbc-all-1.5.0.jar) does not contain any Hadoop classes and if you look at other profiles (default-hadoop, cdh, hdp) in this pom, they all have this in test scope.